src/Package/Impl/Shell/IApplicationShell.cs

```diff
@@ -10,15 +9,5 @@ namespace Microsoft.VisualStudio.R.Package.Shell {
     /// composition container, export provider, global VS IDE
     /// services and so on.
     /// </summary>
-    public interface IApplicationShell : IEditorShell {
-        /// <summary>
-        /// Retrieves global service from the host application.
-        /// This method is not thread safe and should not be called 
-        /// from async methods.
-        /// </summary>
-        /// <typeparam name="T">Service interface type such as IVsUIShell</typeparam>
-        /// <param name="type">Service type if different from T, such as typeof(SVSUiShell)</param>
-        /// <returns>Service instance of null if not found.</returns>
-        T GetGlobalService<T>(Type type = null) where T : class;
-    }
+    public interface IApplicationShell : IEditorShell { }
```

Comment:
Probably not unless we'll need something VS specific. But this will be the next round. ICoreShell (basic things), IUiShell (+UI related methods), and _maybe_ editor part. The latter only translates command targets so I am thinking about turning it into a service.